plone/app/versioningbehavior/__init__.py

```diff
@@ -1,22 +1,20 @@
 # -*- coding: utf-8 -*-
-from plone.namedfile.interfaces import HAVE_BLOBS
 from Products.CMFCore.permissions import ManagePortal
-
 from zope.i18nmessageid import MessageFactory
 
+
 _ = MessageFactory('plone')
 
 
 def initialize(context):
     """Registers modifiers with zope (on zope startup).
     """
-    if HAVE_BLOBS:
-        from modifiers import modifiers
+    from modifiers import modifiers
```

Comment:
if modifiers is a p.a.versioningbehavior module, I would rather prefer a full import and move it to the top now that there is no longer an ``if HAVE_BLOBS``